tmt/log.py

```diff
@@ -605,3 +605,29 @@ def fail(
                 'shift': shift
                 }
             )
+
+    _bootstrap_logger: Optional['Logger'] = None
+
+    @classmethod
+    def get_boostrap_logger(cls) -> 'Logger':
```

Comment:
@happz Could we extend the unit tests for this class function maybe, my only comment, otherwise it looks good

Reply:
Added. There's not much to test though, on top of what we already test, but it runs through the general logger unit test round.